packages/components/src/Checkbox/Checkbox.test.tsx

```diff
@@ -39,6 +40,37 @@ it("renders each variation of checked, defaultChecked and indeterminate", () =>
   });
 });
 
+it("renders a description when set", () => {
+  const { getByText } = render(<Checkbox description="Checkers" />);
+  const description = getByText("Checkers");
+  expect(description).toBeInstanceOf(HTMLParagraphElement);
+});
+
+it("allows react components to be passed as children", () => {
+  const handleLinkClick = jest.fn();
+  const { getByText } = render(
+    <Checkbox description="Checkers">
+      <Text>
+        I agree to the
+        <a
+          style={{
+            color: "var(--color-interactive)",
+            textDecoration: "underline",
+          }}
+          tabIndex={0}
+          onClick={handleLinkClick}
+        >
+          Terms of Service
+        </a>
+      </Text>
+    </Checkbox>,
+  );
+  const TOSAnchorElement = getByText("Terms of Service");
+  expect(TOSAnchorElement).toBeInTheDocument();
+  fireEvent.click(TOSAnchorElement);
+  expect(handleLinkClick).toHaveBeenCalled();
+});
+
```

Comment:
also, could you add more tests to cover more cases? for example: 
1. passing in a regular element and the test should pass (already done 👍 )
2. passing in some random things, (e.g. plain text)
3. passing in some invalid values, (not sure what cases will be considered as invalid for this change, maybe plain text 👆 ?)
4. also some edge cases should be covered, e.g. passing in an empty element and so on

I didn't look super close to this, so some of my suggestions above may or may not make sense. Just use it as general guidance when you add more tests

Reply:
The invalid use case seems hard to test. I've looked at other components that accept children components and none of them seem to go that far either. I can definitely add plain text and an empty element though, thanks for the feedback!

Reply:
I just double checked the tests in `Modal.test.tsx` actually. For testing children it only has one test "modal shows the children and a close button". Do we need to update the tests for Modal to cover more cases, or are we getting out of hand with testing possible children?

Reply:
I feel that we just need to care about this change and cover as many as possible cases for our new changes. I had another thought about adding empty component tests and other tests, I think you can just add the tests that you think make sense to you!

Reply:
I understand. I liked your edge case test, so I added the test for an empty component. We can't do any tests with passing invalid input though - thanks to the magic of Typescript!